### PR TITLE
porter 1.1.0 (new formula)

### PR DESCRIPTION
**Pull request by SpiffyEight77**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/p/porter.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    cd "cmd/porter" do
+      ldflags = %W[
+        -s
+        -X get.porter.sh/porter/pkg.Version=#{version}
+        -X get.porter.sh/porter/pkg.Commit=#{tap.user}
+      ]
+
+      system "go", "build", *std_go_args(ldflags:)
+      generate_completions_from_executable(bin/"porter", "completion")
+      system bin/"porter", "mixin", "install", "exec", "--version", "v#{version}"
```
Comment (branchvincent) [inline Formula/p/porter.rb:21]: is this installing a binary? we'll need to install it from source, is it just https://github.com/getporter/porter/tree/main/pkg/exec?

**Review by branchv — CHANGES_REQUESTED** [review]
thanks, looks pretty close

File: Formula/p/porter.rb (hunk 1 above)
Comment (SpiffyEight77) [reply Formula/p/porter.rb]: > is this installing a binary? we'll need to install it from source, is it just https://github.com/getporter/porter/tree/main/pkg/exec?

Thank you for your review.

Based on the [document](https://porter.sh/docs/getting-started/install-porter/#mixins), we'll need to install it from the source.

Comment (branchvincent) [reply Formula/p/porter.rb]: Also, it looks like this installs to `~/.porter`? Nothing outside the prefix will end up in the bottle, so we either can't do this or need to find a way for porter to find it within `brew --prefix porter`

Comment (SpiffyEight77) [reply Formula/p/porter.rb]: @branchvincent 

Thank you for the information. It appears that we cannot change the `mixin` installation directory in this version, so I have removed the `mixin install` command. We will update the brew installation documentation accordingly for users.

Comment (branchvincent) [reply Formula/p/porter.rb]: sounds good

**Review by branchv — APPROVED** [review]
thanks for your contribution! 🍻

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
@SpiffyEight77, thanks for your contribution to Homebrew! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request merged**